helpers/sherpa_config.py

```diff
@@ -33,25 +33,25 @@ class sherpa_config(Command):
     description = "Configure Sherpa build options. If in doubt, ignore this command and stick to defaults. See setup.cfg for more information."
     user_options = [
                     ('fftw', None, "Whether Sherpa should build the embedded fftw3 library, which is the default behavior: set to 'local' to make Sherpa link against existing libraries on the system.)"),
-                    ('fftw-include-dirs', None, "Where the fftw3 headers are located, if fftw is 'local'"),
-                    ('fftw-lib-dirs', None, "Where the fftw3 libraries are located, if fftw is 'local'"),
-                    ('fftw-libraries', None, "Name of the libraries that should be linked as fftw3"),
+                    ('fftw_include_dirs', None, "Where the fftw3 headers are located, if fftw is 'local'"),
```

Comment:
So, it appears that this change is only an issue for the `setup.cfg` file, and not here, so it is technicaly not necessary, **but**

- I can't find any obvious documentation in `setuptools` that talks about this change (the best I saw was "go and read about it at https://setuptools.pypa.io/en/latest/deprecated/index.html" which at the moment has nothing obvious about this change
- I have been confused about this before - e.g. #1452 - so making all things consistent is probably a good idea

Reply:
Yes, I don't think it's required, but I decided that - if we do changes already - then that's a good opportunity to be more consistent. The Inconsistency has caused confusion in the past.